config.js

```diff
@@ -6,4 +6,5 @@ config.redis_server_port = 6379;
 config.redis_server_name = '127.0.0.1';
 config.default_retry_interval = 10; // in 10 seconds
 config.att_convert_retry_interval = 60; // in 60 seconds for transmog
+config.no_retry_services = "value";
```

Comment:
Well - the "annoying" part is that without this value - node breaks on the app for me. Which is weird - how does it even work at all?
Also - if we don't have it in the config, we probably don't care about the value...
But I'll change that to an array.